arcade/decorator_alternative.py

```diff
@@ -0,0 +1,63 @@
+import arcade
+
+__all__ = ["override", "decorator_run"]
+
+_registry = {}
+
+
+class ArcadeContextError(AssertionError):
```

Comment:
Not too sure on the name of this, or whether a custom error is required. Alternatives?